test/e2e/subscription_e2e_test.go

```diff
@@ -1218,6 +1197,18 @@ var _ = Describe("Subscription", func() {
 				require.FailNowf(GinkgoT(), "failed", "subscription contains unexpected installplan condition: %v", cond)
 			case operatorsv1alpha1.SubscriptionInstallPlanMissing:
 				require.Equal(GinkgoT(), operatorsv1alpha1.ReferencedInstallPlanNotFound, cond.Reason)
+			case operatorsv1alpha1.SubscriptionCatalogSourcesUnhealthy:
+				newCond := sub.Status.GetCondition(condType)
+				require.Equal(GinkgoT(), cond.Status, newCond.Status)
+				require.Equal(GinkgoT(), cond.Message, newCond.Message)
+				// The Reason check:
+				// CatalogSources[Added|Deleted] could overwrite AllCatalogSourcesHealthy, but leave Status/Message preserved
+				// Added/Deleted could subesequently be overwritten by Healthy
+				// This all depends on when reconciliation occurs and if the number of catalog sources changed between reconcilliations
```

Comment:
```suggestion
				// This all depends on when reconciliation occurs and if the number of catalog sources changed between reconciliations
```

Reply:
Ugh... no spell checker in my code editor!